rls-analysis/src/analysis.rs

```diff
@@ -155,6 +184,48 @@ impl PerCrateAnalysis {
             None => false,
         }
     }
+
+    // Returns all identifiers which overlap with `span`. There is no guarantee about
+    // the ordering of identifiers in the result, but they will probably be roughly
+    // in order of appearance.
+    #[cfg(feature = "idents")]
+    fn idents(&self, span: &Span) -> Vec<Ident> {
+        self.idents
+            .get(&span.file)
+            .map(|by_line| {
+                (span.range.row_start..=span.range.row_end)
```

Comment:
Since it seems that anything idents related is lazily calculated; if  we want to make it usable on stable without an additional feature we could do something like
```rust
(span.range.row_start.0..=span.range.row_end.0).map(Row::<ZeroIndexed>::new)...
```

Reply:
We could but I'd like to see motivation for it first (i.e., somebody using stable wants to use this feature).